Templating/Helper/DateTimeHelper.php

```diff
@@ -166,6 +166,12 @@ public function getDatetime($data, $timezone = null)
             return $data;
         }
 
+        if ($data instanceof \DateTimeImmutable) {
+            $date = \DateTime::createFromFormat('U', $data->getTimestamp());
```

Comment:
Thoughtless OTV : https://a-nickels-worth.blogspot.fr/2016/04/a-guide-to-naming-variables.html